clubManagement/urls.py

```diff
@@ -1,9 +1,10 @@
 from django.conf.urls import url
 
-from clubManagement.views import AttendanceAddView, YearAttendanceReportView
+from clubManagement.views import AttendanceAddView, YearAttendanceReportView, MonthAttendanceReportView
 
 urlpatterns = [
     url(r'^attendance-report/(?P<year>[0-9]+)/(?P<user_id>[0-9]+)/$', YearAttendanceReportView.as_view(), name='attendance_report_day'),
+    url(r'^attendance-report/(?P<year>[0-9]+)/(?P<month>[0-9]+)/(?P<user_id>[0-9]+)/$', MonthAttendanceReportView.as_view(), name='attendance_report_month'),
```

Comment:
r'^attendance-report/(?P<batch>[0-9]+)/(?P<year>[0-9]+)/(?P<month>[0-9]+)/$'

Make the url like this. Access this variables in the get() using kwargs.get('batch') ...